plugins/debug.c

```diff
@@ -1046,8 +1052,13 @@ static void draw_reg_map(int valid_register)
 	for (i = 0; i < p_reg->bgroup_cnt; i++){
 		p_bit = &p_reg->bgroup_list[i];
 		for (j = (p_bit->width - 1); j >= 0 ; j--){
-			gtk_container_add(GTK_CONTAINER(hboxes[p_bit->offset]), lbl_bits[p_bit->offset + j]);
-			gtk_box_reorder_child((GtkBox *)hboxes[p_bit->offset], lbl_bits[p_bit->offset + j], -1);
+			parent = gtk_widget_get_parent(GTK_WIDGET(lbl_bits[p_bit->offset + j]));
+			gtk_widget_hide(GTK_WIDGET(lbl_bits[p_bit->offset + j]));
+			g_object_ref(GTK_WIDGET(lbl_bits[p_bit->offset + j]));
```

Comment:
when removing a widget from a container, the object is unrefed by the function gtk_container_remove. we need to add a ref to the widget before removing it from its current container, in order to be able to add it to the new one. i suppose a suitable place to unref it would be when closing the plugin

Reply:
added unref after putting widget in new container